src/app/pages/HomePage/Graph/Graph/index.tsx

```diff
@@ -699,9 +699,16 @@ const GraphCmp: ForwardRefRenderFunction<SVGSVGElement, GraphProps> = (
           {...handleHover(Layer.consensus, setHoveredLayer)}
         >
           {(isMobile || hoveredLayer !== Layer.consensus) && (
-            <text x="161" y="212" fill={graphTheme.text} fontSize="12px">
+            <GraphLayerStatus
```

Comment:
1) it's misaligned for me can we adjust `textY` ?
2) Currently it works the same way as removed line. It never shows online/offline badge after clicking Consensus circle.

Reply:
> It never shows online/offline badge after clicking Consensus circle.

![image](https://github.com/oasisprotocol/explorer/assets/2093792/beca5375-c350-44a4-a7a8-981749e721ac)

Reply:
> it's misaligned for me can we adjust textY ?

OK, after adjustment:

![image](https://github.com/oasisprotocol/explorer/assets/2093792/ddb3a8bd-49aa-4904-80d2-232aad339396)

Is it OK now?

Reply:
> It never shows online/offline badge after clicking Consensus circle.

The trick is that it only show the badge when we are zoomed in, but clicking the Consensus circle doesn't really zoom in, since it still selects the same network. The only case we actually see this badge if we zoom in by clicking on another circle, and the panning to the consensus circle.

I was not sure if this was the desired behavior, so I asked @donouwens to check, he did, and said that it works OK the way it is.

Reply:
ahh if it's by design ignore my comment

Reply:
Don says it has to zoom in :shrug:

Reply:
See follow-up PR https://github.com/oasisprotocol/explorer/pull/1404